app/views/hosts/edit.html.erb

```diff
@@ -12,5 +12,6 @@
      )
   %>
 <% end %>
-
+<%# The next hidden div is a workaround for host redirect after a submit %>
+<div class='hidden' id='hidden-host-name'><%= @host.name %></div>
```

Comment:
We do not need a special field for this, we have a @host.name in a host[name] field :)

Reply:
tried that but the `host[name]` is just the name without the domain suffix